packages/core/src/app/order/OrderSummaryItem.tsx

```diff
@@ -35,26 +35,27 @@ const OrderSummaryItem: FunctionComponent<OrderSummaryItemProps> = ({
         </figure>
 
         <div className="product-column product-body">
-            <h5
+            <h4
```

Comment:
Before: we use `h1`, `h2`, `h3`, `h5`. So it was complained that `checkout-js` does not provide appropriate heading levels in the page and maintain hierarchy of the headings.

After: we use  `h1`, `h2`, `h3`, `h4`.